app/services/shopify_import/importers/base_importer.rb

```diff
@@ -1,16 +1,7 @@
 module ShopifyImport
   module Importers
     class BaseImporter
-      def self.import!(**params)
```

Comment:
Let's add those when we actually use them? Right now I see we're adding a lot of functionality that we _might_ use, like base.new with both client and credentials parameters, params/options passing around, etc.

I agree we should think about future needs, like re-importing and importing from multiple stores, but I don't think we should focus on providing functionality for that step right now. Having DataStore where we store what has been added is good enough. From there we can build on and have other extra functionalities. So I'd first focus on having "one full import working end-to-end". Adding params here later on will be extremely easy.